README.md

```diff
@@ -5,29 +5,31 @@
 
 The Rust-ML working group is currently at an early stage of development, and is still deciding on several choices at an organizational level. This document will be updated accordingly as those decisions are made.
 
-### How can you contribute?
+## What do we do?
 
-Come say hello, read about, and weigh in on our [discussions](https://github.com/rust-ml/discussion)! We currently have several ongoing threads regarding upcoming scope of work, organizational structures, and more.
+In general, the Rust-ML working group's goals are to:
 
-The Rust-ML group has a Zulip channel. Feel free to take a look and say hi!
+- Create spaces where people interested in doing machine learning in Rust can meet, ask questions, and discuss their work
+- Help organize support for Rust libraries, tools, and projects in the machine learning domain
+- In rare instances, to assist with or assume the maintainership of projects important to the wider community
```

Comment:
Maybe reword? I think I would like to foster Rust help to ML experts, the way this is worded might drive them off slightly?

Reply:
What about an additional bullet point, something like `- Create a mature production ready machine learning ecosystem`

Reply:
This is definitely the weakest area of the change, and I think it's worth some additional discussion. During the video meeting today, @deg4uss3r mentioned something along the lines of helping people with machine learning become familiar with Rust, and enabling people familiar with Rust to do machine learning. 

@xd009642 This is definitely an area for discussion about WG goals. For example, the Embedded Group very clearly directly develops crates and tooling, whereas the GameDev WG explicity does not build things like games/game engines, only fills the gaps in lower-level tools (https://github.com/rust-gamedev/wg/blob/master/charter.md). I'm unsure where along that spectrum we should sit, but I just trying to be wary of coming across in a way that says "hey, this WG has *the* way of doing machine learning in Rust"

Reply:
Hmm I'd say both working groups are working towards ecosystems just a bit more high level versus low level. What about "Begin to shape the developing rust machine learning ecosystem".

Reply:
Also I'll endeavour to turn up to the meeting next week as long as I have no clashes :smile:

Reply:
Hmm, that might work. Let me make the other changes, and then we can maybe take this in particular to the woodshed a little bit more

Reply:
I've made the other change, but this is still an area that I think could use some more work